# Call for Abstracts

The CFDE Steering Committee invites you to submit an abstract of your scientific research or technical work to be considered for a poster or oral presentation during the CFDE meeting, **April 4-5**, 2023 at the Bethesda Hotel.


Abstracts can be of a scientific or technical nature. Abstracts should provide clear data and validated conclusions. They should consist of an introduction that presents the background and rationale of the study or work, a brief description of the methods employed, a summary of the results, and a statement of the conclusions, best practices, or lessons learned.

Because this meeting is a confidential, internal "study" meeting, abstracts submitted to other national/international conferences are eligible and welcomed for co-submission. 

## Poster Size and Layout

Poster boards are 6 ft wide x 4 ft high. Posters can be in portrait or landscape mode but should not excede the poster stand dimensions. Space for posters and time for presentations will be limited. If the organizers of the CFDE meeting receive more abstracts than there is available space, priority will be given to those abstracts that were submitted in a timely fashion and that provide all the information that is requested. We will also be selecting a small subset of poster presenters to give brief lightning-style talks at the start of the poster session (2 min or less).

## Dates

The poster reception taking place in the afternoon of Tuesday, April 4th. 

The CFDE will confirm poster and oral presentation assignments with the presenting authors by **March 28, 2023**.

## [Abstract Submission Form](https://docs.google.com/forms/d/e/1FAIpQLSdyKekDI4tnoDdj9XWGwab6N83DTtlqaNXAP3T5CvjOO7g9Kw/viewform)

Please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdyKekDI4tnoDdj9XWGwab6N83DTtlqaNXAP3T5CvjOO7g9Kw/viewform) to submit your sbtract.

**SUBMISSIONS ARE LIMITED TO TWO ABSTRACTS PER PRESENTER**

## Abstract Submissions

1. **CFDE-GSKG: Gene Set Integration across Common Fund Datasets**<br>John Erol Evangelista(1), Daniel J. B. Clarke(1), Zhuorui Xie(1), Avi Ma’ayan(1)<br>(1) Data Coordination and Integration Center (DCIC) for the Library of Network-based Cellular Signatures (LINCS), Department of Pharmacological Sciences, Mount Sinai Center for Bioinformatics, Icahn School of Medicine at Mount Sinai, New York, NY 10029<br><br>NIH Common Fund (CF) programs have accelerated transformative science by producing massive omics datasets for the research community. A challenge remains in making these data more Findable, Accessible, Interoperable, and Reusable (FAIR), and integrating these datasets to aid researchers in querying data across CF programs. To address this challenge here we introduce the Common Fund Data Ecosystem (CFDE) Gene Set Knowledge Graph (GSKG), a knowledge graph database and web-based application that serves gene sets extracted from CF programs datasets to provide an integrated enrichment analysis tool. So far, nine gene set libraries were created by processing datasets from LINCS, GTEx, Metabolomics, IDG, GlyGen, IMPC, and HuBMAP. To use CFDE-GSKG, gene sets obtained from omics experiments by individual investigators can be submitted to the tool using a simple input form. The resulting subgraph produced by CFDE-GSKG illuminates connections between the input gene set and various CF gene sets that overlap with the queried gene set. Such analysis enables users to explore Common Fund datasets for further hypothesis generation for their research. Besides enrichment analysis, CFDE-GSKG can be used to query the consolidated CF knowledge by searching for cross-CF-dataset-associations for a gene or a biological term, for example, a drug, a metabolite, a tissue, a cell type, or a phenotype. This feature of CFDE-GSKG discovers connections between entities that may not be obvious. Future releases of CFDE-GSKG would extend the database to serve data from more Common Fund programs, as well as add predicted associations with graph-based machine learning methods, opening the doors to a broader cross Common Fund knowledge discovery resource. The CFDE-GSKG prototype is freely available at: https://cfde-gskg.dev.maayanlab.cloud/.   

1. **Exploring the Crosstalk between Disease Pathways in Kidneys from Patients with HIV and Diabetic Nephropathy with CFDE Tools**<br>Zhuorui Xie (1), Daniel J. B. Clarke (1), John Erol Evangelista (1), Alexander Lachmann (1), Juncheng Pan (1), Zhengzi Yi (2), Wenlin Wang (2), Weijia Zhang (2), John Cijiang He (2), Avi Ma’ayan (1)<br>1. Department of Pharmacological Sciences, Icahn School of Medicine at Mount Sinai, New York, NY; 2. Division of Nephrology, Department of Medicine, Icahn School of Medicine at Mount Sinai, New York, NY<br><br>The prolonged exposure to combination antiretroviral therapies (cART) for patients living with HIV promotes the development of diabetes. This increases the prevalence of diabetic kidney disease (DKD) in this patient population. However, the molecular mechanisms of DKD in patients living with HIV are mostly unknown. To address this knowledge gap, RNA-seq profiles from kidney biopsies were obtained from healthy controls, patients with diabetic nephropathy (DN) alone, and patients with DN who are also HIV positive. To analyze this dataset, we utilized the CFDE Playbook Partnership Workflow Builder (PPWB), an interactive web-based visual interface that facilitates workflow construction by connecting microservices provided by various Common Fund (CF) Data Coordination Centers (DCCs). In addition, gene sets extracted from the processed RNA-seq data were analyzed with the CFDE Gene Set Knowledge Graph (CFDE-GSKG) prototype app. Analysis and visualization using the CFDE-PPWB and CFDE-GSKG tools uniquely provide novel insights about the molecular differences in the kidney of healthy controls, DKD-HIV-, and DKD-HIV+ patients. Selected hypotheses generated from this analysis are expected to be tested in cell-based assays and animal models of the disease. 

1. **MetGENE: Gene-centric Metabolomics Information Retrieval Tool**<br>Sumana Srinivasan, Mano Maurya,  Srinivasan Ramachandran, Eoin Fahy and Shankar Subramaniam<br>Department of Bioengineering, University of California San Diego, San Diego, CA<br><br>Background: Biomedical research often involves contextual integration of multi-modal and multi-omic data in search of mechanisms for improved diagnosis, treatment and monitoring. Researchers need to access information from diverse sources comprising data in various and sometimes incomplete formats. The downstream processing of the data, to decipher mechanisms by reconstructing networks and developing quantitative models, warrants considerable effort. <br>Methods: It is designed as a web-based application using PHP and JavaScript as the front-end and at the back-end, R scripts with wrapper functions to retrieve information from various data repositories such as KEGG  (for reaction and metabolite/compound<br>IDs) and Metabolomics Workbench (for metabolite RefMet names<br>and study IDs).  MetGENE maintains session variables for species ID and organism name; ENTREZ gene ID and gene symbol; anatomy, disease and phenotype terms, and the previous values of these terms to enable server-side caching of pages and thus avoid unnecessary and time-consuming fetching of data across the network. For programmatic ease of access, we provide REST APIs that convert each table of the information displayed into its<br>corresponding JSON format. The REST APIs are developed using Smart/OpenAPI format API .<br>Results: MetGENE is a knowledge-based, gene-centric data aggregator that hierarchically retrieves information about the gene(s), their related pathway(s), reaction(s),metabolite(s), and metabolomic studies from standard data repositories under one dashboard to enable<br>ease of access through centralization of relevant information. Further, the information can be contextualized by filtering along species, anatomy (tissue) and condition (disease or phenotype).  MetGENE is an open source tool and is available at:<br>https://bdcw.org/MetGENE/index.php. 

1. **Systematic Personalized Cell Surface Target and Drug Identification from KidsFirst RNA-seq Profiling of Tumors** <br>Giacomo B. Marino (1), Eden Z. Deng (1), Adam C. Resnick (2), Avi Ma’ayan (1)<br>1. Data Coordination and Integration Center (DCIC) for the Library of Network-based Cellular Signatures (LINCS), Department of Pharmacological Sciences, Mount Sinai Center for Bioinformatics, Icahn School of Medicine at Mount Sinai, One Gustave L. Levy Place, Box 1603, New York, NY 10029, USA;<br>2. Center for Data Driven Discovery in Biomedicine, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA; Division of Neurosurgery, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA<br><br>The KidsFirst NIH Common Fund program has characterized the molecular signatures of thousands of diverse pediatric brain tumors. These patients have limited treatment options with poor survival rates. One of the most promising emerging approaches to treat these patients is by targeting tumors with immunotherapies such as antibody-drug conjugates (ADCs) and CAR T-cell therapies. However, the initial challenge with developing such immunotherapies is identifying targets and drugs that may effectively eliminate cancer cells while leaving healthy cells and tissues unharmed. By combining data collected by other Common Fund (CF) programs together with the transcriptomics and proteomics data collected by KidsFirst, we can identify and prioritize targets and drugs that may be applied as therapies tailored to the individual patient. To achieve this, we compared KidsFirst pediatric cancer expression data to normal tissues and cell types expression data from GTEx, Tabula Sapiens, ARCHS4, and HuBMAP to identify genes that are significantly upregulated in the tumors, lowly expressed across normal tissues and cell types, and encode cell surface proteins. At the same time, we also prioritized drugs and pre-clinical small-molecule compounds using the most recent LINCS L1000 data for the same patients. The workflows to perform such analyses are made available via the CFDE-supported bioinformatics web-based applications: TargetRanger and SigCom LINCS. TargetRanger is available from https://targetranger.maayanlab.cloud/; and SigCom LINCS is available from https://maayanlab.cloud/sigcom-lincs/. These bioinformatics workflows and web apps enabled the recovery of previously identified targets and small molecules, as well as several novel targets and drugs for further experimental exploration. Overall, the entire framework produces hypotheses for individualized treatments for pediatric cancer patients by integrating data from the CF programs KidFirst, LINCS, HuBMAP and GTEx.
